docs/install.md

```diff
@@ -3,22 +3,50 @@
 
-MLRun comprises of two parts: MLRun Server and MLRUN client.
+MLRun comprises two parts: MLRun Server and MLRUN client.
 
-## Deployment options
+**In this section**
+- [Deployment options](#deployment-options)
+- [Set up your client](#Set up your client)
+- [Security context](#security-context)
+- [MLRun client backward compatibility](#MLRun client backward compatibility)
 
+## Deployment options
 
 There are several deployment options:
 - [Local deployment](https://docs.mlrun.org/en/latest/install/local-docker.html): Deploy a Docker on your laptop or on a single server.
-   This option is good for testing the waters or when working in a small scale environment. It's limited in terms of computing resources and scale, but
-   simpler for deployment.
-
+   This option is good for testing the waters or when working in a small scale environment. It's limited in terms of computing resources 
+   and scale, but simpler for deployment.
 - [Kubernetes cluster](https://docs.mlrun.org/en/latest/install/kubernetes.html): Deploy an MLRun server on Kubernetes.
-   This option deploys MLRun on a Kubernetes cluster, which supports elastic scaling. Yet, it is more complex to install as it requires you to install Kubernetes on your own.
-  
-- [Iguazio's Managed  Service](https://www.iguazio.com): A commercial offering by Iguazio. This is the fastest way to explore the full set of MLRun functionalities.<br>
+   This option deploys MLRun on a Kubernetes cluster, which supports elastic scaling. Yet, it is more complex to install as it requires 
+   you to install Kubernetes on your own.
+  - [Iguazio's Managed  Service](https://www.iguazio.com): A commercial offering by Iguazio. This is the fastest way to explore 
+  the full set of MLRun functionalities.<br>
   Note that Iguazio provides a 14 day free trial.
+  
+## Security context (root vs. non-root user)(#security-context)
 
+By default, MLRun assigns the root user to MLRun runtimes and pods. You can improve the security context by changing the security mode, 
+which is implemented by Igauzio during installation, and applied system-wide:
+- Override: Use the user id of the user that triggered the current run or use the nogroupid for group id. Requires Iguazio v3.5.1.
+- Disabled: Security context is not auto applied (the system aplies the root user). (default)
+
+If your system is configured in disabled mode, you can apply the security context to individual runtimes/pods by using `function.with_security_context`, and the job is assigned to the user or to the user's group that ran the job.<br>
+(You cannot override the user of individual jobs if the system is configured in override mode.) The options are:
+
+```from kubernetes import client as k8s_client
+
+security_context = k8s_client.V1SecurityContext(
+            run_as_user=1000,
+            run_as_group=3000,
+        )
+function.with_security_context(security_context)
+```
+ Services that do not support security context: 
```

Comment:
Should say what "do not support" means, I believe in these cases the security context provided will just be ignored - I'm sure @Tankilevitch can provide the details here.

Reply:
I think we can separate the services to two:
1. Services created by mlrun 
   - Kaniko
   - Spark services
3. Infrastructure services 
   - Kubeflow pipelines core services
  
Kaniko isn't something we expose to the user, meaning we need to explain when kaniko will be created.
Suggestion:
- Kaniko - Used for building images ( can avoid using it by providing prebuilt images which contain all the requirements )

 Regarding @theSaarco comment, I think we should state that for the services created by mlrun those two specific services are not being enriched with security context and add explanation for each, so in kaniko is the reason we are waiting for kaniko 1.9.0 to be released which add support for that.

Reply:
Looks like Kaniko 1.9.0 was already released. Do we want to say it will be supported by MLRun in the near future?
Spark 3.2 is also released. Same? will be supported by MLRun in the near future?